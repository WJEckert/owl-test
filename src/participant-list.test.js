import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ParticipantList from './participant-list.js';

it('renders without crashing', () => {
    const div = document.createElement('section');
    ReactDOM.render(<ParticipantList participants =
         {[{id : 1, inSession : true, onStage : true, avatar : 'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1', name : 'Will'}]}/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  
  it('renders the UI as expected', () => {
    const tree = renderer
    .create(<ParticipantList participants =
        {[{id : 1, inSession : true, onStage : true, avatar : 'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1', name : 'Will'}]}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });