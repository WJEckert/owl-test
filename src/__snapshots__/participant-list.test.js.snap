// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<section
  className="list"
>
  <div
    className="list-cycle"
  >
    <div
      className="participant"
    >
      <div
        className="image"
      >
        <img
          alt="Picture of person"
          className="participant-image"
          src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
        />
      </div>
      <div
        className="info"
      >
        <h2>
          Ashla Attwool
        </h2>
        <ul
          className="listOn"
        >
          <li>
             on stage
          </li>
        </ul>
      </div>
    </div>
    <div
      className="participant"
    >
      <div
        className="image"
      >
        <img
          alt="Picture of person"
          className="participant-image"
          src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
        />
      </div>
      <div
        className="info"
      >
        <h2>
          Frank Runciman
        </h2>
        <ul
          className="listOn"
        >
          <li>
             in session
          </li>
        </ul>
      </div>
    </div>
    <div
      className="participant"
    >
      <div
        className="image"
      >
        <img
          alt="Picture of person"
          className="participant-image"
          src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
        />
      </div>
      <div
        className="info"
      >
        <h2>
          Caty Flucker
        </h2>
        <ul
          className="listOn"
        >
          <li>
             in session
          </li>
        </ul>
      </div>
    </div>
    <div
      className="participant"
    >
      <div
        className="image"
      >
        <img
          alt="Picture of person"
          className="participant-image"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
      </div>
      <div
        className="info"
      >
        <h2>
          Koren Templeton
        </h2>
        <ul
          className="listOn"
        >
          <li>
             on stage
          </li>
        </ul>
      </div>
    </div>
    <div
      className="participant"
    >
      <div
        className="image"
      >
        <img
          alt="Picture of person"
          className="participant-image"
          src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
        />
      </div>
      <div
        className="info"
      >
        <h2>
          Axe Kubicka
        </h2>
        <ul
          className="listOff"
        >
          <li>
             left session
          </li>
        </ul>
      </div>
    </div>
  </div>
</section>
`;
